index.src.html

```diff
@@ -4686,11 +4693,21 @@
     11. Let |decodeResult| be a new {{ImageDecodeResult}} initialized as
         follows:
         1. Assign 'true' to {{ImageDecodeResult/complete}}.
-        2. Let |timestamp| and |duration| be the presentation timestamp and
-            duration for |output| as described by |encodedFrame|. If
-            |encodedFrame| does not describe a timestamp or
-            duration, assign `null` to the corresponding variable.
-        3. Assign {{ImageDecodeResult/image}} with the result of running the
+        2. Let |timestamp| be the presentation timestamp for |output| as
+            described by |encodedFrame|. If |encodedFrame| does not describe a
+            timestamp, assign `0` to |timestamp|.
+
+            NOTE: For animated images, the timestamp of a given frame will
+                generally equal the sum of durations from preceeding frames.
+                However, implementers are not expected to decode earlier frames
+                to compute this timestamp. If it is infeasible to dermine a
+                imestamp by simply inspecing |encodedFrame|, timestamp can
+                instead be `0`.
```

Comment:
This should in fact be normative, not a note, which are non-normative. Probably `SHOULD`: the implementation is supposed to provide a timestamp, but can decide not to do it for good reasons (here, because it's a waste of resources for some codecs/image formats). This also means that an implementation that puts in 0 timestamp is conformant.